server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -283,8 +275,11 @@ public Version minimumCompatibilityVersion() {
         if (major == 6) {
             // force the minimum compatibility for version 6 to 5.6 since we don't reference version 5 anymore
             return Version.fromId(5060099);
-        } else if (major >= 7) {
-            // all major versions from 7 onwards are compatible with last minor series of the previous major
+        } else if (major == 7) {
+            // force the minimum compatibility for version 7 to 6.8 since we don't reference version 6 anymore
+            return Version.fromId(6080099);
```

Comment:
I don't think we need this. master should not care about the compatibility version of a previous version, only with itself. However, given the precedent of it existing for 6.x->5.6 above, I'm ok with this going in for now (I will propose an alternative in a followup).

Reply:
@rjernst thank you for reviewing. I would like to handle follow-up. 
Can you please share issues once created?